projects/services-layers/src/lib/types/Layers.ts

```diff
@@ -178,7 +187,7 @@ export class Layer implements ILayerOptions {
   /** a layer might have more than one style; eg. true color and false color for the same dataset */
   styles?: ILayerStyleSet[];
   /** The crossOrigin attribute for loaded images if you want to access pixel data with the Canvas renderer */
-  crossOrigin?: string;
+  crossOrigin?: CrossOriginType;
```

Comment:
If this attribute is able for both (IRasterLayerOptions/IVectorLayerOptions) please remove it from them and add it to the general ILayerOptions because both are extended from it.